pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -17,29 +17,30 @@ limitations under the License.
 package v1alpha1
 
 import (
-	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
```

Comment:
Jeff has a task to write a pr to kubernetes/community to make this easier for the next person that has to do it.